model/config.go

```diff
@@ -2306,49 +2306,49 @@ func (s *LocalizationSettings) SetDefaults() {
+	IdpDescriptorUrl            *string `access:"authentication_saml"` // telemetry: none
+	IdpMetadataUrl              *string `access:"authentication_saml"` // telemetry: none
+	ServiceProviderIdentifier   *string `access:"authentication_saml"` // telemetry: none
+	AssertionConsumerServiceURL *string `access:"authentication_saml"` // telemetry: none
 
-	SignatureAlgorithm *string `access:"authentication"`
-	CanonicalAlgorithm *string `access:"authentication"`
+	SignatureAlgorithm *string `access:"authentication_saml"`
+	CanonicalAlgorithm *string `access:"authentication_saml"`
 
-	ScopingIDPProviderId *string `access:"authentication"`
-	ScopingIDPName       *string `access:"authentication"`
+	ScopingIDPProviderId *string `access:"authentication_saml"`
+	ScopingIDPName       *string `access:"authentication_saml"`
 
-	IdpCertificateFile    *string `access:"authentication"` // telemetry: none
-	PublicCertificateFile *string `access:"authentication"` // telemetry: none
-	PrivateKeyFile        *string `access:"authentication"` // telemetry: none
+	IdpCertificateFile    *string `access:"authentication_saml"` // telemetry: none
+	PublicCertificateFile *string `access:"authentication_saml"` // telemetry: none
+	PrivateKeyFile        *string `access:"authentication_saml"` // telemetry: none
 
 	// User Mapping
-	IdAttribute          *string `access:"authentication"`
-	GuestAttribute       *string `access:"authentication"`
+	IdAttribute          *string `access:"authentication_saml"`
+	GuestAttribute       *string `access:"authentication_saml"`
 	EnableAdminAttribute *bool
 	AdminAttribute       *string
-	FirstNameAttribute   *string `access:"authentication"`
-	LastNameAttribute    *string `access:"authentication"`
-	EmailAttribute       *string `access:"authentication"`
-	UsernameAttribute    *string `access:"authentication"`
-	NicknameAttribute    *string `access:"authentication"`
-	LocaleAttribute      *string `access:"authentication"`
-	PositionAttribute    *string `access:"authentication"`
+	FirstNameAttribute   *string `access:"authentication_saml"`
+	LastNameAttribute    *string `access:"authentication_saml"`
+	EmailAttribute       *string `access:"authentication_saml"`
+	UsernameAttribute    *string `access:"authentication_saml"`
+	NicknameAttribute    *string `access:"authentication_saml"`
+	LocaleAttribute      *string `access:"authentication_saml"`
+	PositionAttribute    *string `access:"authentication_saml"`
 
-	LoginButtonText *string `access:"authentication"`
+	LoginButtonText *string `access:"authentication_saml"`
 
-	LoginButtonColor       *string `access:"authentication"`
-	LoginButtonBorderColor *string `access:"authentication"`
-	LoginButtonTextColor   *string `access:"authentication"`
+	LoginButtonColor       *string `access:"authentication_saml"`
+	LoginButtonBorderColor *string `access:"authentication_saml"`
+	LoginButtonTextColor   *string `access:"authentication_saml"`
```

Comment:
These are the same as above, keep them belonging to experimental features.  They should be
```
	LoginButtonColor       *string `access:"experimental_features"`
	LoginButtonBorderColor *string `access:"experimental_features"`
	LoginButtonTextColor   *string `access:"experimental_features"`
```